fuel-merkle/src/common/msb.rs

```diff
@@ -5,11 +5,11 @@ pub enum Bit {
 }
 
 trait GetBit {
-    fn get_bit(&self, bit_index: usize) -> Option<Bit>;
+    fn get_bit(&self, bit_index: u32) -> Option<Bit>;
```

Comment:
Are these changes to `fuel-merkle` necessary? It feels like we're changing library logic to fit business logic. A library should promise correctness under all targets it supports, including all 32 bit and 64 bit targets. If it has internal problems with truncation, that should be fixed in the library. But this change appears to be changing the library interface to accommodate changes to external libraries.

Reply:
It is a private trait, so it is an internal detail of how Merkle tree works=)